packages/Tag/src/Tag.styles.js

```diff
@@ -1,66 +1,10 @@
 import styled, { css } from "styled-components";
 import tokens from "@paprika/tokens";
-import Button from "@paprika/button";
+import RawButton from "@paprika/raw-button";
```

Comment:
btw @paprika/button can be removed from the package.json if we switched to @paprika/raw-button